src/nvim/CMakeLists.txt

```diff
@@ -401,10 +401,16 @@ endif()
 
 set(NVIM_EXEC_LINK_LIBRARIES ${NVIM_LINK_LIBRARIES} ${LUA_PREFERRED_LIBRARIES})
 
+# Enable LTO (Link Time Optimization) #8654
 if(POLICY CMP0069)
   cmake_policy(SET CMP0069 NEW)
 endif()
 
+# Link libraries by full path even in implicit directories. #10407
+if(POLICY CMP0060)
+  cmake_policy(SET CMP0060 NEW)
+endif()
```

Comment:
This appears to make no difference really - or at least is not enough by itself.